openrtb_ext/imp_aduptech.go

```diff
@@ -0,0 +1,11 @@
+package openrtb_ext
+
+import "encoding/json"
+
+type ExtImpAduptech struct {
+	Publisher string          `json:"publisher"`
+	Placement string          `json:"placement"`
+	Query     string          `json:"query"`
+	AdTest    string          `json:"adtest"`
```

Comment:
Can we use `bidrequest.test` instead of introducing a separate parameter, considering it is a standard field already used by Prebid?

Reply:
Hi,

Unfortunately, we already have this parameter in the Prebid JS Adapter.
See: https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/aduptech.md?plain=1#L26